CMakeLists.txt

```diff
@@ -53,6 +53,12 @@ install(DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/ DESTINATION ${CMAKE_INSTALL
 install(DIRECTORY pd-help/ DESTINATION ${CMAKE_INSTALL_PREFIX})
 
 #Dependencies: availability
+#Non-standard openssl path on OSX
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
```

Comment:
I just had the time to do some reading on the OpenSSL issue.
In short, it's an issue of the current MACOS as Apple decided to ship an old OpenSSL that is marked as deprecated and thus removed the header file while keep the linker configuration available.
Thus, the header is not found and a newer version of OpenSSL can be installed in parallel while linking will fail without further configuration as the original SSL version will be used.
My understanding is that Apple will remove OpenSSL completely in future version and thus resolve the introduced issue: 
https://solitum.net/openssl-os-x-el-capitan-and-brew/

Thus, this fix is only required for El Capitan and I don't think it is good style to add a workaround for this (obviously the OS distributor should fix this).